x-pack/plugins/security_solution/scripts/endpoint/resolver_generator.ts

```diff
@@ -95,19 +95,25 @@ async function main() {
     eventIndex: {
       alias: 'ei',
       describe: 'index to store events in',
-      default: 'events-endpoint-1',
+      default: 'logs-endpoint.events.fake-default',
+      type: 'string',
+    },
+    alertIndex: {
+      alias: 'ai',
+      describe: 'index to store alerts in',
+      default: 'logs-endpoint.alerts-default',
       type: 'string',
     },
     metadataIndex: {
       alias: 'mi',
       describe: 'index to store host metadata in',
-      default: 'metrics-endpoint.metadata-default-1',
+      default: 'metrics-endpoint.metadata-default',
```

Comment:
did something change in epm or elastic to make this possible?

Reply:
the `-1` was a temporary thing until we started using data streams. Data streams now add the `000001` portion, so just as clean up I was moving everything to be the expected values now.